### PR TITLE
Bump runtime to 21.08 and update Sky Checkers to 1.3

### PR DESCRIPTION
**Pull request by bbhtt**
- Update shared modules
- Update SDL2 to 2.0.20

Ping @zorgiepoo, please review when you have time, thanks!

### CODE REVIEW

File: net.zgcoder.skycheckers.json [hunk 1]
```diff
@@ -54,9 +54,13 @@
 			"sources": [
 				{
 					"type": "archive",
-					"url": "https://github.com/zorgiepoo/Sky-Checkers/releases/download/1.2/sc-snap-pak-repo.tgz",
-					"sha256": "ff0a0e7a608e52a3b8b1d8d7fadfc05a953b34142e5fc5af943f2e1b1bd2057d"
-				}
+					"url": "https://github.com/zorgiepoo/Sky-Checkers/archive/refs/tags/v1.3.tar.gz",
+					"sha256": "b9ed99216def4e4b5d3609f016f9be2502b0734db4d8548af62ffb2517668d68"
```
Comment (zorgiepoo) [inline net.zgcoder.skycheckers.json:58]: I think I need to create a `sc-snap-pak-repo.tgz` for 1.3 otherwise full git repo is overkill.

Comment (bbhtt) [reply net.zgcoder.skycheckers.json:58]: Yea sure, if you add it in a release I can drop the release patch.

**Comment by zorgiepoo** [comment]
Thanks for the PR. I would like to do some testing and perhaps more thorough review. I assume the external motivation for updating this is to update the runtime and dependencies..? I don't think 1.3 made any changes for Linux from 1.2 so I just haven't updated it there. But to do this in a more clean way I probably have to make another release, but then the other platforms I support don't need an update..

**Comment by bbhtt** [comment]
> I assume the external motivation for updating this is to update the runtime and dependencies..? I don't think 1.3 made any changes for Linux from 1.2 so I just haven't updated it there. 

Yes. 

If you want I can downgrade it back to 1.2 or will you make a new release? I'm fine with either.

**Comment by hfiguiere** [comment]
Runitme 19.08 is EOL by end of August, that's the main reason.

File: net.zgcoder.skycheckers.json (hunk 1 above)
Comment (bbhtt) [reply net.zgcoder.skycheckers.json]: Changed to sc-snap-pak-repo.tgz for 1.3

**Comment by bbhtt** [comment]
Keeping this in sync with https://github.com/zorgiepoo/Sky-Checkers/commit/aa2040c96ec18639df605ebfbc088da91ca04d28

**Comment by zorgiepoo** [comment]
This looks good so I'll merge this. Thank you for helping / updating this. I'm keeping SDL at 2.0.20 for now as later releases have a fullscreen-toggle bug in my game I need to investigate at some later point..

**Pull request merged**